<!DOCTYPE html>
<html lang="en">
  <head>
    <meta http-equiv="content-type" content="text/html; charset=utf-8" />
    <meta name="google-site-verification" content="f_--N3WuBtfzUzdq05UpgO98Vy6YeBD9qdJ0v6geVBg" />
	<title>Martin Jinye Zhang's Homepage</title>

    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <!-- Le styles -->
    <link href="assets/css/bootstrap.min.css" rel="stylesheet">
    <link href="assets/css/bootstrap-responsive.min.css" rel="stylesheet">
    <link href="assets/css/feixia.css" rel="stylesheet">

    <!-- Le HTML5 shim, for IE6-8 support of HTML5 elements -->
    <!--[if lt IE 9]>
    <script src="assets/js/html5shiv.js"></script>
    <![endif]-->
  </head>

  <body>

    <div class="container">
      <div class="row">
        <div class="span3 bs-docs-sidebar">
          <!-- We use a fancy nav bar if there is enough space -->
          <hr class="hidden-phone"/>
          <ul class="nav nav-list bs-docs-sidenav hidden-phone">
            <li><a href="#"><i class="icon-play"></i>Home</a></li>     
			<li><a href="#position & education"><i class="icon-play"></i>Position & Education</a></li>
			<li><a href="#papers"><i class="icon-play"></i>Papers</a></li>
			<li><a href="#softwares"><i class="icon-play"></i>Softwares </a></li>
			<li><a href="#professional services"><i class="icon-play"></i>Professional services</a></li>
			<li><a href="#honors"><i class="icon-play"></i>Honors</a></li>
            		<li><a href="assets/pdf/martin_zhang_resume.pdf"><i class="icon-play"></i>CV</a></li>
			
		  </ul>
          <hr class="hidden-phone"/>
          <div class="text-center hidden-phone">
            <img src="assets/img/mz_face.jpeg" alt="photo" class="logo-image"/>
          </div>
          <!-- Otherwise, we simply use a flat list of links -->
          <div class="visible-phone">
            <a href="#"><i class="icon-play"></i>Home</a>
		<a href="#education"><i class="icon-play"></i>Education</a>
            <a href="#manuscripts"><i class="icon-play"></i>Manuscripts</a>
            <a href="#publications"><i class="icon-play"></i>Publications</a>
            <a href="#professional services"><i class="icon-play"></i>Professional services</a>
            <a href="#honors"><i class="icon-play"></i>Honors</a>
		<li><a href="#softwares"><i class="icon-play"></i>Softwares </a></li>
            <a href="assets/pdf/martin_zhang_resume.pdf"><i class="icon-play"></i>CV</a>
          </div>
        </div>
        <div class="span9">
          <h3>
	Martin Jinye Zhang
</h3>
<h5>
martinjzhang at gmail.com

<a href="https://github.com/martinjzhang">[Github]</a>
<a href="https://scholar.google.com/citations?user=zjr6n-QAAAAJ&hl=en">[Google Scholar]</a>
<a href="https://twitter.com/martinjzhang">[Twitter]</a>

</h5>

<p>
	Hello! I am Martin, a postdoc at Harvard School of Public Health working with Prof. <a href="https://www.hsph.harvard.edu/alkes-price/">Alkes Price</a> on statistical genetics. Currently, I am interested in analyzing the UK biobank whole exome sequencing (WES) data, and potential methods for combining GWAS with single-cell RNA-seq. Before that, I did my PhD at Stanford with Prof. <a href="http://web.stanford.edu/~dntse/">David Tse</a> and Prof. <a href="https://sites.google.com/site/jamesyzou/">James Zou</a> on statistics, machine learning, and computational biology. Some topics include empirical Bayes, multiple hypothesis testing, multi-armed bandits, single-cell RNA-seq. 
<p>
<p>
	I publish under <b>Martin Jinye Zhang</b> (and its variations). I also go under Jinye Zhang (张金野).
<p>


<!--My research focuses on the development of statistical methods for the inference in various high-throughput genetic data applications like GWAS, RNA-seq, single-cell RNA-seq, and multi-omics. I am particularly interested in methods that systematically aggregate different datasets, algorithm acceleration via adaptive sampling, and empirical Bayes modeling.-->

<div>
	<b><font color="#8C1515">News:</font> </b>	

	<ul>
		<li><p> 2019.12 Our <a href="https://www.biorxiv.org/content/10.1101/2019.12.23.887604v1">paper</a> on identifying aging signatures using the Tabula Muris Senis data is available online.
	</ul>

	<ul>
		<li><p> 2019.11 Our <a href="https://www.biorxiv.org/content/10.1101/389296v2">paper</a> on experimental design for single-cell RNA-seq was accepted by <font color="#0000ff"> Nature Communications</font>. I also contributed to the <a href="https://www.biorxiv.org/content/10.1101/661728v2">Tabular Muris Senis</a> paper revision.
	</ul>
	<ul>
		<li><p> 2019.9 I joined Harvard T.H. Chan School of Public Health as a postdoc. I will work with Prof. <a href="https://www.hsph.harvard.edu/alkes-price/">Alkes Price</a> on statistical genetics.
	</ul>
	<ul>
		<li><p> 2019.6 Our <a href="https://www.biorxiv.org/content/early/2018/12/13/496372">AdaFDR</a> paper on covariate-adaptive multiple testing was accepted by <font color="#0000ff"> Nature Communications</font>.
	</ul>	

<!--
	<ul>
		<li><p> 2019.5 Our <a href="https://www.biorxiv.org/content/early/2018/12/13/496372">AdaFDR</a> paper on covariate-adaptive multiple testing received the <b style="color: rgb(204,0,0);"> RECOMB Best Paper Award</b>.</p></li>		
	</ul>	
	<ul>
		<li><p> 2019.4 The <a href="https://arxiv.org/abs/1902.00197">adaptive Monte Carlo multiple testing</a> paper got accepted by <font color="#0000ff"> ICML 2019</font>. I will work with Prof. <a href="https://www.hsph.harvard.edu/alkes-price/">Alkes Price</a> as a postdoctoral researcher at Harvard T.H. Chan School of Public Health starting from Sept. 2019. Our paper "Longitudinal multi-omics of host–microbe dynamics in prediabetes" was accepted by <font color="#0000ff">Nature</font>.	 		
	</ul>

	<ul>
		<li><p> 2019.4 I will work with Prof. <a href="https://www.hsph.harvard.edu/alkes-price/">Alkes Price</a> as a postdoctoral researcher at Harvard T.H. Chan School of Public Health starting from Sept. 2019		
	</ul>
	<ul>
		<li><p> 2019.4 I will give a talk at RECOMB 2019 on <a href="https://www.biorxiv.org/content/10.1101/496372v1">AdaFDR</a>, a covariate-adaptive approach for multiple hypothesis testing.</p></li>		
	</ul>

	<ul>
		<li><p> 2019.4 Our paper "Longitudinal multi-omics of host–microbe dynamics in prediabetes" was accepted by <i>Nature</i>. </p></li>		
	</ul>

	<ul>
		<li><p> 2019.3 I will give a talk at CISS 2019 on <a href="https://www.biorxiv.org/content/10.1101/389296v2">optimal single-cell RNA-seq experiment design</a>.</p></li>		
	</ul>
	<ul>
		<li><p> 2019.2 I will give a talk at ITA 2019 on <a href="https://arxiv.org/abs/1902.00197">adaptive Monte Carlo multiple testing</a> that accelerates the standard workflow of permutation test + BH multiple testing by 1000x.</p></li>		
	</ul>
	<ul>
		<li><p> 2018.12 Our <a href="https://www.biorxiv.org/content/early/2018/12/13/496372">AdaFDR</a> paper on covariate-adaptive multiple testing got accepted by RECOMB 2019 as top 5% best paper!</p></li>		
	</ul>
</div>
-->


<h3>Position & Education</h3>
<a name="position & education"></a>
<div class="media">
    <span class="pull-left"><img src="./assets/img/hsph1.png" width="80px" height="96px"/></span>
    <div class="media-body">
        <p><span style="font-weight: bold">2019 -  Present</span>, T.H. Chan School of Public Health, <i><b>Harvard University</b></i>,</p>
        <p> Postdoctoral Researcher</p>
    </div>
</div>

<div class="media">
    <span class="pull-left"><img src="./assets/img/stanford.png" width="80px" height="96px"/></span>
    <div class="media-body">
        <p><span style="font-weight: bold">2014 - 2019</span>, Department of Electrical Engineering, <i><b>Stanford University</b></i>,</p>
        <p>Doctor of Philosophy (PhD)</p>
</div>
</div>

<div class="media">
    <span class="pull-left"><img src="./assets/img/stanford.png" width="80px" height="96px"/></span>
    <div class="media-body">
        <p><span style="font-weight: bold">2014 - 2017</span>, Department of Electrical Engineering, <i><b>Stanford University</b></i>,</p>
        <p>Master of Science (MS)</p>
</div>
</div>

<div class="media">
    <span class="pull-left"><img src="./assets/img/tsinghua.png" width="80px" height="96px"/></span>
    <div class="media-body">
        <p><span style="font-weight: bold">2010 - 2014</span>, Department of Electronic Engineering, <i><b>Tsinghua University</b></i>,</p>
        <p>Bachelor of Engineering (B.Eng.)</p>
    </div>
</div>


<!--
<h3>Selected projects</h3>
<a name="project"></a>

<h4>Adaptive Monte Carlo computation</h4>
 <p> Many computational problems can be reformulated as statistical problems with uniform sampling, where adaptive sampling may significantly improve the computational efficiency. The adaptive sampling procedure can usually be designed via multi-armed bandits. </p>
<div class="media">
    <span class="pull-left"><img src="./assets/img/meddit.png" width="240px" height="240px"/></span>
    <div class="media-body">

	<p><b>Adaptive Monte Carlo Multiple Testing via Multi-Armed Bandits</b> <br> <i>ICML 2019</i> (joint work with James Zou and David Tse) <a href="https://arxiv.org/abs/1902.00197">[pdf]</a> <a href="https://github.com/bagavi/Meddit">[code]</a></p>
	<p>We consider multiple testing with Monte Carlo p-values. Here, first n MC samples are used to computed a Monte Carlo p-value for each null hypothesis and then the Benjamini-Hochberg procedure is applied on the set of Monte Carlo p-values to control false positives. Noticing that not all p-values need to be estimated with the same accuracy and therefore with the same number of MC samples, we propose AMT (adaptive Monte Carlo multiple testing) that adaptively estimates the MC p-values and control false positives. AMT accelerates the workflow by 1000x in GWAS setting, reducing the computation time from 2 months to 1 hour.</p>

	<p><b>Medoids in Almost Linear Time via Multi-armed Bandits</b> <br> <i>AISTATS 2018</i> (joint work with Vivek Bagaria<sup>*</sup>, Govinda Kamath<sup>*</sup>, Vasilis Ntranos<sup>*</sup> and David Tse) <a href="https://github.com/bagavi/Meddit">[code]</a></p>
        <p> Computing the medoid of a large number of points in high-dimensional space is an increasingly common operation in many data science problems. We present an algorithm Med-dit to compute the medoid with high probability, which uses O(nlogn) distance evaluations. Med-dit is based on a connection with the Multi-Armed Bandit problem. We evaluate the performance of Med-dit empirically on the Netflix-prize and single-cell RNA-seq datasets, containing hundreds of thousands of points living in tens of thousands of dimensions, and observe a 5-10x improvement in performance over the current state of the art. </p>
    </div>
</div>


<h4>Covariate-adaptive multiple testing</h4>
<div class="media">
    <span class="pull-left"><img src="./assets/img/adafdr.png" width="560px" height="400px"/></span>
    <div class="media-body">
	<p><b>AdaFDR: a Fast, Powerful and Covariate-Adaptive Approach to Multiple Hypothesis Testing</b> <br>accepted as  Cell Systems best paper in <i>RECOMB 2019</i>, received <b style="color: rgb(204,0,0);"> RECOMB Best Paper Award</b>, published in <i>Nature Communications</i>, 2018 (joint work with Fei Xia and James Zou) <a href="https://www.biorxiv.org/content/early/2018/12/13/496372">[pdf]</a> <a href="https://github.com/martinjzhang/adafdr">[software]</a> <a href="https://github.com/martinjzhang/AdaFDRpaper">[code to reproduce the paper]</a></p> 
	<p><b>NeuralFDR: learning decision threshold from hypothesis features</b> <br><i>NeurIPS 2017</i> (joint work with Fei Xia<sup>*</sup>, James Zou, David Tse) <a href="https://github.com/fxia22/NeuralFDR">[code]</a></p>
        <p> </p>
    </div>
</div>

<p> Many studies in computational biology have a relatively small sample size and thus have limited detection power. This can be addressed by utilizing knowledge from previous studies. Such idea is explored here in the context of multiple hypothesis testing, whose goal is to maximize the number of discoveries while controlling the fraction of false discoveries. In many settings, in addition to the p-value, additional information/covariates for each hypothesis are available. For example, in eQTL studies, each hypothesis tests the correlation between a variant and the expression of a gene. We also have additional covariates such as the location, conservation and chromatin status of the variant, which could inform how likely the association is to be due to noise. However, popular multiple hypothesis testing approaches, such as Benjamini-Hochberg procedure (BH) and independent hypothesis weighting (IHW), either ignore these covariates or assume the covariate to be univariate. We develop covariate-adaptive multiple testing methods that adaptively learns the optimal p-value threshold from covariates to significantly improve detection power. 
</p>

<h4>Experimental design for single-cell RNA-seq</h4>
<div class="media">
    <span class="pull-left"><img src="./assets/img/oneread.png" width="240px" height="240px"/></span>
    <div class="media-body">
	<p><b>One read per cell per gene is optimal for single-cell RNA-seq</b> <br>under review in <i>Nature Communications</i>, 2018 (joint work with Vasilis Ntranos<sup>*</sup> and David Tse) <a href="https://www.biorxiv.org/content/early/2018/08/09/389296">[pdf]</a> <a href="https://github.com/martinjzhang/single_cell_eb">[code]</a></p>
        <p>An underlying question for virtually all single-cell RNA sequencing experiments is how to allocate the limited sequencing budget: deep sequencing of a few cells or shallow sequencing of many cells? A mathematical framework reveals that, for estimating many important gene properties, the optimal allocation is to sequence at the depth of one read per cell per gene. Interestingly, our analysis shows that the corresponding optimal estimator is not the widely-used plug-in estimator but one developed via empirical Bayes. </p>
    </div>
</div>
-->
		
		
<h3>Manuscripts</h3>
<a name="manuscripts"></a>
<ul>
    <li><p><font color="#0f5ddb"> Mouse Aging Cell Atlas Analysis Reveals Global and Cell Type Specific Aging Signatures. </font> 
	<a href="https://www.biorxiv.org/content/10.1101/2019.12.23.887604v1">[pdf]</a> <a href="https://github.com/czbiohub/tabula-muris-senis/tree/master/2_aging_signature">[code]</a> <br>
	<b>Martin Jinye Zhang</b>, Angela Oliveira Pisco, Spyros Darmanis, James Zou.<br>
	<i> Under review in <font color="#0000ff"> eLife </font>(2019).</I>
	</p></li> 
</ul>

<ul>
    <li><p><font color="#0f5ddb"> A Single Cell Transcriptomic Atlas Characterizes Aging Tissues in the Mouse. </font> 
	<a href="https://www.biorxiv.org/content/10.1101/661728v2">[pdf]</a> <a href="https://github.com/czbiohub/tabula-muris-senis">[code]</a> <br>
	The Tabula Muris Consortium.<br>
	<i> Under review in <font color="#0000ff"> Nature </font>(2019).</I> Contributed to differential expression analysis (Fig. 2f,g,i,j,l,m) and cluster diversity score (Fig. 4 c-f).
	</p></li> 
</ul>
(*Equal contribution)

<h3>Publications</h3>
<a name="publications"></a>
<ul>
    <li><p><font color="#0f5ddb"> Adaptive Monte Carlo Multiple Testing via Multi-Armed Bandits. </font> 
	<a href="https://arxiv.org/abs/1902.00197">[pdf]</a> <a href="https://github.com/martinjzhang/AMT">[code]</a> <br>
	<b>Martin J. Zhang</b>, James Zou, David Tse.<br>
	<i > <font color="#0000ff"> ICML </font>(2019).</i>  
	</p></li> 
</ul>

<ul>
    <li><p><font color="#0f5ddb"> Fast and covariate-adaptive method amplifies detection power in large-scale multiple hypothesis testing. </font> <a href="https://github.com/martinjzhang/adafdr">[software]</a> <a href="https://github.com/martinjzhang/AdaFDRpaper">[code to reproduce paper]</a> <br>
	<b>Martin J. Zhang</b>, Fei Xia, James Zou.<br>
	<i> <font color="#0000ff"> Nature Communications </font>(2019).</I> Preliminary version accepted as the Cell Systems best paper in <i>RECOMB 2019</i> and received the <b style="color: rgb(204,0,0);"> RECOMB Best Paper Award</b> <br>
	(Preliminary version: "AdaFDR: a Fast, Powerful and Covariate-Adaptive Approach to Multiple Hypothesis Testing". <a href="https://www.biorxiv.org/content/early/2018/12/13/496372">[pdf])</a> <br>  
	</p></li> 
</ul>

<ul>
    <li><p><font color="#0f5ddb"> Determining sequencing depth in a single-cell RNA-seq experiment. </font> 
	<a href="">[pdf]</a> <a href="https://github.com/martinjzhang/single_cell_eb">[code]</a> <br>
	<b>Martin J. Zhang<sup>*</sup></b>, Vasilis Ntranos<sup>*</sup>, David Tse.<br>
	<i> <font color="#0000ff"> Nature Communications </font>(2019).</i> <br>
	(Preliminary version: "One read per cell per gene is optimal for single-cell RNA-seq". <a href="https://www.biorxiv.org/content/early/2018/08/09/389296">[pdf])</a> <br>

	</p></li> 
</ul>

<ul>
    <li><p><font color="#0f5ddb"> Longitudinal multi-omics of host–microbe dynamics in prediabetes. </font> <br>
	Wenyu Zhou<sup>*</sup>, M. Reza Sailani<sup>*</sup>, Kévin Contrepois<sup>*</sup>, Yanjiao Zhou<sup>*</sup>, Sara Ahadi<sup>*</sup>, Shana Leopold, <b>Martin J. Zhang</b>, ..., George M. Weinstock, Michael Snyder.<br>
	<i> <font color="#0000ff"> Nature </font>(2019).</I> Contributed 3 panels in 2 figures.
	</p></li> 
</ul>

<ul>
    <li><p><font color="#0f5ddb"> Exploring Patterns Unique to a Dataset with Contrastive Principal Component Analysis. </font> 
	<a href="https://github.com/abidlabs/contrastive">[code]</a> <br>
	Abubakar Abid<sup>*</sup>, <b>Martin J. Zhang<sup>*</sup></b>, Vivek K. Bagaria, James Zou.<br>
	<i> <font color="#0000ff"> Nature Communications </font>(2018).</I>
	</p></li> 
</ul>

<ul>
    <li><p><font color="#0f5ddb"> Medoids in Almost Linear Time via Multi-armed Bandits. </font> 
	<a href="https://github.com/bagavi/Meddit">[code]</a> <br>
	Vivek Bagaria<sup>*</sup>, Govinda Kamath<sup>*</sup>, Vasilis Ntranos<sup>*</sup>, <b>Martin J. Zhang<sup>*</sup></b>, David Tse.<br>
	<i> <font color="#0000ff"> AISTATS </font>(2018).</I>
	</p></li> 
</ul>

<ul>
    <li><p><font color="#0f5ddb"> NeuralFDR: learning decision threshold from hypothesis features. </font> 
	<a href="https://github.com/fxia22/NeuralFDR">[code]</a> <br>
	Fei Xia<sup>*</sup>, <b>Martin J. Zhang<sup>*</sup></b>, James Zou, David Tse.<br>
	<i> <font color="#0000ff"> NeurIPS </font>(2017).</I>
	</p></li> 
</ul>

<ul>
    <li><p><font color="#0f5ddb"> Block-wise MAP Inference for the Determinantal Point Processes with Application to Change Point Detection. </font> <br>
	<b>Martin J. Zhang</b>, Zhijian Ou.<br>
	<i> <font color="#0000ff"> SSP </font>(2016).</I>
	</p></li> 
</ul>

<ul>
    <li><p><font color="#0f5ddb"> On the Theoretical Analysis of Cross Validation in Compressive Sensing. </font> <br>
	<b>Jinye Zhang</b>, Laming Chen, Petros T. Boufounos, and Yuantao Gu.<br>
	<i> <font color="#0000ff"> ICASSP </font>(2014).</I>
	</p></li> 
</ul>

(*Equal contribution)

<h3>Softwares</h3>
<a name="softwares"></a>
<ul>
    <li><p><a href="https://github.com/martinjzhang/adafdr">adafdr</a>: covariate-adaptive multiple testing.
	</p></li> 
</ul>

<ul>
    <li><p><a href="https://github.com/martinjzhang/single_cell_eb">sceb</a>: sequencing-depth aware estimators for single-cell RNA-seq analysis via empirical Bayes.
	</p></li> 
</ul>

<ul>
    <li><p><a href="https://github.com/bagavi/Meddit">Meddit</a>: an almost linear algorithm for computing the medoid for a set of n points via adaptive sampling.
	</p></li> 
</ul>

<ul>
    <li><p><a href="https://github.com/abidlabs/contrastive">contrastive</a>: a python library for performing unsupervised machine learning on datasets with learning (e.g. PCA) in contrastive settings, where one is interested in patterns (e.g. clusters or clines) that exist one dataset, but not the other.
	</p></li> 
</ul>

<h3>Professional services</h3>
<a name="professional services"></a>
<ul>
<li> Journal paper reviewer for <font color="#0000ff"> Nature Communications </font>, <font color="#0000ff"> Bioinformatics </font>, <font color="#0000ff"> Journal of Genetics and Genomics </font></li>
<li> Conference paper reviewer for <font color="#0000ff"> NeurIPS 2019 </font>, <font color="#0000ff"> NeurIPS 2016 </font></li>
<li> Organizer for the <a href="http://isl.stanford.edu/colloquium.html">Information Systems Laboratory Colloquium</a>, 2015-2019, EE, Stanford. </li>
</ul>

<h3>Honors and Awards</h3>
<a name="honors"></a>
<ul>
<li> 2019 <font color="#0000ff"> RECOMB 2019 </font>  <a href="assets/img/RECOMB_best_paper.jpeg">best paper award</a> </li>
<!--
<li> 2019 <font color="#0000ff"> RECOMB 2019 </font> best paper award </li>
-->
<li> 2019 <font color="#0000ff"> RECOMB 2019 </font> travel award </li>
<!--
<li> 2018 Cell Systems Best Papers of <i>RECOMB 2019</i> </li>
-->
<li> 2017 <font color="#0000ff"> NeurIPS 2017 </font> travel award </li>
<li> 2015 Stanford Graduate Fellowship (SGF, Inventec Fellow) </li>
<li> 2015 Numerical Technologies Award in Electrical Engineering (Numerical Technologies Founders Graduate Fellowship) </li>
<li> 2015 Ranked 2/79 in the EE PhD Qualifying Exam at Stanford University </li>
<li> 2014 Outstanding Undergraduate Thesis "Speech Diarization Based on the Determinantal Point Processes" at Tsinghua University
<li> 2013 Comprehensive Excellence Scholarship in Electronic Engineering at Tsinghua University </li>
<!--
<li> 2011 First award in Beijing College Student Physics Competition, 3/186 in Department of Electronic Engineering at Tsinghua University </li>
<li> 2009 First Prize Provincial and Bronze Medal National, Chinese Physics Olympiad </li>
-->
</ul>





<h3>Teaching Experiences</h3>
<a name="honors"></a>
<ul>
<li> TA, EE 278: Introduction to Statistical Signal Processing (Spring 2017)  </li>
</ul>


<!--
<h3>Press Coverage</h3>
<a href = "https://www.genomeweb.com/scan/week-genome-research-29"><img src="./assets/img/gw.png" width="80px"/> </a>
<a href = "https://medium.com/syncedreview/is-it-real-or-is-it-gibson-a-new-virtual-environment-for-ai-training-da588a707370"><img src="./assets/img/synced.png"  width="60px"/> </a>
-->

<!--
<h3>Softwares</h3>
-->

<!--
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-80823373-1', 'auto');
  ga('send', 'pageview');

</script>-->

<!-- <script type="text/javascript" id="clustrmaps" src="//cdn.clustrmaps.com/map_v2.js?d=GB91RaxcjbQAH78D8ZqC4CrkB4id3lS6DT4kESYsRTU&cl=ffffff&w=a"></script> -->





        </div>
      </div>
    </div>

    <!-- Footer
    ================================================== -->
    <hr>
    <footer class="footer">
    <div class="container">
      <div class="row">
        <div class="span12">
          <p>&copy; Martin J. Zhang 2018-2020 
	     <a class="pull-right" href="#"><script type='text/javascript' id='clustrmaps' src='//cdn.clustrmaps.com/map_v2.js?cl=ffffff&w=150&t=n&d=GB91RaxcjbQAH78D8ZqC4CrkB4id3lS6DT4kESYsRTU'></script></a>
		  </p>
        </div>
      </div>
    </div>
    </footer>





    <!-- Le javascript
    ================================================== -->
    <!-- Placed at the end of the document so the pages load faster -->
<!--
    <script type="text/javascript" src="http://platform.twitter.com/widgets.js"></script>
    <script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
    <script src="assets/js/bootstrap-transition.js"></script>
    <script src="assets/js/bootstrap-alert.js"></script>
    <script src="assets/js/bootstrap-modal.js"></script>
    <script src="assets/js/bootstrap-dropdown.js"></script>
    <script src="assets/js/bootstrap-scrollspy.js"></script>
    <script src="assets/js/bootstrap-tab.js"></script>
    <script src="assets/js/bootstrap-tooltip.js"></script>
    <script src="assets/js/bootstrap-popover.js"></script>
    <script src="assets/js/bootstrap-button.js"></script>
    <script src="assets/js/bootstrap-collapse.js"></script>
    <script src="assets/js/bootstrap-carousel.js"></script>
    <script src="assets/js/bootstrap-typeahead.js"></script>
    <script src="assets/js/bootstrap-affix.js"></script>
    <script src="assets/js/holder/holder.js"></script>
    <script src="assets/js/application.js"></script>
-->
  </body>
</html>
